### PR TITLE
BookStore Styles

### PR DESCRIPTION
**Pull request by luccamicale**
## I added the styles in my BookStore 💯 👍 

##https://github.com/luccamicale/bookstore 🥇 

## Checklist before requesting a review 💯 🔢 
- [x] I have performed a self-review of my code
- [x] I follow the instructions about styles
- [x] I understood how to implement different styles
- [x] I keep my code very tidy and clean 👍

### CODE REVIEW

File: src/modules-css/addbook.css [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .form {
-  border: 1px solid rgb(197, 186, 186);
-  width: 87.3%;
+  border: 1px solid whitesmoke;
+  width: 91.3%;
```
Comment (tufoinnkuo10) [inline src/modules-css/addbook.css:3]: - [x] great job.the add book title and author is not visible as it is right down and kind of hiden. please increase the bottom margin so it follows the zeplin design.
Screenshot (592)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @luccamicale 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by luccamicale** [comment]
I modified the change that ask me the last reviewer. Now My project is complete and perfect!!!
It should be approved!! 💯

**Review by BenMukebo — APPROVED** [review]
### Hi @luccamicale, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.
- [OPTIONAL] It will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. Please add descriptive steps from cloning your app to seeing the results displayed on the browser.. This includes `git clone URL` , `cd project_name`,  `npm install`, `npm start`, etc 🙃 


- [OPTIONAL] When I click the link to the `MIT.md` file that you added, I get this 404 `error`:

00Mit


#### The reason for this is that there is no `MIT.md` file in the root directory of this project. Kindly attach the file. 👍

- [OPTIONAL] The styling has to be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). Please add the missing vertical bars and make sure to increate  your input author form  width as shown on the picture ✍🏻 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**